merlin/dataloader/loader_base.py

```diff
@@ -653,6 +619,115 @@ def _pack(self, gdf):
             return gdf
         return gdf.toDlpack()
 
+    @property
+    def schema(self) -> Schema:
```

Comment:
I add this for backwards compatibility in case we're using the `loader.schema` anywhere. Would be good to remove this once we're sure it's not being used anywhere though.